core/web_admin.py

```
@@ -0,0 +1,1040 @@
+    async def login_reset(self) -> dict[str, Any]:
+        await self._discard_login_attempt(remove_files=True)
+        self._login_data = {
+            "replace_existing": True,
+            "need_password": False,
+            "created_at": datetime.now().isoformat(timespec="seconds"),
+        }
+        return {"message": "已进入重新登录流程，当前已登录账号会保留到新账号登录成功。"}
+
+    async def runtime_check(self) -> dict[str, Any]:
+        self.plugin.forwarder._stopping = False
+        asyncio.create_task(self.plugin.forwarder.check_updates())
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)